packages/web-wallet/src/App.tsx

```diff
@@ -19,7 +20,7 @@ function App() {
   }, [location.pathname]);
 
   useInterval(() => {
-    triggerRescan();
+    if(installedSnap) triggerRescan();
```

Comment:
You can put this logic directly inside triggerRescan method as we have an access to the state